kaos/utils/time_conversion.py

```diff
@@ -0,0 +1,49 @@
+"""This module contains all date/time related conversion code."""
+
+import datetime
+import calendar
+from astropy.time import Time
+
+def utc_to_unix(time_string):
+    """Takes a string input of the format 'YYYYMMDDTHH:MM:SS.MS' and converts it to a UNIX
```

Comment:
Copying comment from #16: Add a clarification here that the 'T' is the actual letter T